application.py

```diff
@@ -38,7 +38,7 @@ def call_matlab(d):
                    d["speed"], composition["o"], composition["o2"],
                    composition["n2"], composition["he"], composition["h"],
                    ACCOM_MODEL_DICT[d["accommodationModel"]],
-                   d["energyAccommodation"], d["surfaceMass"], [[]], "",
+                   d["energyAccommodation"], d["surfaceMass"], matlab.double([]), "",
```

Comment:
this allows singlepoint api to work without changing MAIN.m and prepares for multipoint.

Reply:
Excellent find!